04-store-api/starter/controllers/products.js

```diff
@@ -0,0 +1,67 @@
+const Product = require('../models/product');
+
+const getAllProductsStatic = async (req, res) => {
+  const products = await Product.find({ price: { $gt: 30 } })
+    .select('name price')
+    .limit(10)
+    .skip(5);
+  res.status(200).json({ products, nbHits: products.length });
```

Comment:
> nbHits

Is this name from the course or did you create this name? We need to be careful about abbreviated names like this. My coworker recently said to me in a code review, "are we experiencing a supply chain shortage of letters?" Very funny, but he's right – this name makes sense here, but what about on the client, where the name isn't right next to `products.length`?

How can we solve this problem – what do you think?

Reply:
nbHits is from the course)
I fully agree with your coworker. I should have given more descriptive variable names or at least made some comments about what this code line does so that other developers could read and understand. Regarding ```products``` I could rename it and again give a more descriptive variable name e.g. ```selectedProducts``` or ```selSkipAndLimProducts```.